tests/dragonfly/replication_test.py

```diff
@@ -1836,3 +1836,46 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     await c_replica.close()
     master.stop()
     replica.stop()
+
```

Comment:
we want to test that eviction replicated to replica both from background process and from insert flow
to check that we do enter the eviction in insert flow I suggest setting flag enable_heartbeat_eviction=false and test eviction

Reply:
now the test will have two subtests:
1) test only policy based eviction
2) test both eviction together